### ISSUE TITLE
differenct about ESRGAN and PSNR

### ISSUE THREAD

**Issue by suke27** [xinntao/ESRGAN#8]
hello, I check the 2 pretrained models that you provided, what is difference about 2 models. different training dataset or different loss function.

**Comment by xinntao** [comment]
@suke27 

1. 

`RRDB_PSNR_x4.pth` is the model trained with L1 loss and want to achieve higher **PSNR**.
`RRDB_ESRGAN_x4.pth` is the model trained for high **visually perceptual quality**, with another VGG and GAN loss. 

2. I have not trained x8 models.
